src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -71,6 +74,35 @@ def __init__(self, revision, cache_root, coveralls_token, codecov_token,
 
         logger.info('Mercurial revision', revision=self.revision)
 
+    def get_chunks(self):
+        return list(set([f.split('_')[1] for f in os.listdir('ccov-artifacts')]))
+
+    def get_coverage_artifacts(self, suite=None, chunk=None):
+        files = os.listdir('ccov-artifacts')
+
+        # If suite and chunk are None, return all artifacts.
+        # Otherwise, only return the ones which have suite or chunk in their name.
+        if suite is None and chunk is None:
+            search_substring = ''
```

Comment:
Nit: This feels a bit hacky. Wouldn't it be cleaner to call `search_substring = None` here, and use `if search_substring is None or search_substring in fname:` below?

Reply:
Yes, I will clean this up.